common/test/run-amount.c

```diff
@@ -128,7 +128,6 @@ int main(void)
 	PASS_MSAT(&msat, "1.234567890btc", 123456789000);
 	PASS_MSAT(&msat, "1.2345678901btc", 123456789010);
 	PASS_MSAT(&msat, "1.23456789012btc", 123456789012);
-	FAIL_MSAT(&msat, "1btc");
```

Comment:
why are these cases removed?

Reply:
This belongs to the next commit in which i change the parsing

Reply:
Ah no, they were part of the right commit! Well because it was just added above, and as outlined in the commit message..
>  Allows `.*btc` amounts (without post-decimal)